docs/streams/developer-guide/dsl-api.html

```diff
@@ -3492,7 +3494,6 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
                     </ul>
                 </div>
                 <p>Then we can leverage the <code class="docutils literal"><span class="pre">PopularPageEmailAlert</span></code> processor in the DSL via <code class="docutils literal"><span class="pre">KStream#process</span></code>.</p>
-                <p>In Java 8+, using lambda expressions:</p>
```

Comment:
We dropped support for Java 7 a while ago, so I reckon we can just document Java 8 now. But I didn't want to go on a crusade through the docs either. I touched this section because it's related to my docs task.